pupil_src/shared_modules/surface_tracker/background_tasks.py

```diff
@@ -479,3 +510,48 @@ def _export_gaze_on_surface(self, gazes_on_surface, surface, surface_name):
                                 gp["confidence"],
                             )
                         )
+
+    def _export_fixations_on_surface(self, fixations_on_surf, surface, surface_name):
+        """
+        fixations_on_surf structure: fixations_on_surf[world_frame_idx][event_idx]
+        """
+        with open(
+            os.path.join(
+                self.metrics_dir, "fixations_on_surface" + surface_name + ".csv"
+            ),
+            "w",
+            encoding="utf-8",
+            newline="",
+        ) as csv_file:
+            csv_writer = csv.writer(csv_file, delimiter=",")
+            csv_writer.writerow(
+                (
+                    "fixation_id",
+                    "start_timestamp",
+                    "duration [ms]",
+                    "dispersion [deg]",
```

Comment:
@papr any particular reason to have `[ms]` and `[deg]` here? are there any backwards-compatibility or consistency reasons? I'm asking because all the other labels follow the rules for identifier, so these 2 look out of place a bit.

Reply:
Agreed. I thought it would be nice to add some self-documentation to the headers but you are right that it would be inconsistent. The correct place to put this information is the official documentation.